pkg/sql/lex/BUILD.bazel

```diff
@@ -31,3 +31,42 @@ go_test(
         "//pkg/sql/sessiondatapb",
     ],
 )
+
+go_binary(
```

Comment:
I don't think this is necessary anymore. If you regenerate all bazel files again (you'll need to, `bazel run //:gazelle`), bazel will generate a target for this binary in the new `allkeywords` directory. We'll be able to then refer to it using `$(location //pkg/sql/lex/allkeywords)` I think (or something close to it.

Reply:
Let me give a try